install.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+set -e
+trap 'previous_command=$this_command; this_command=$BASH_COMMAND' DEBUG
```

Comment:
I've just copied the header and the footer from build.sh, though I don't fully understand what it's supposted to do.

Reply:
The main goal was to exit the script if a command fails. And print out the command that failed.

Reply:
No sure if the trap DEBUG is needed though.

Reply:
If I understand https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#index-trap-125 correctly, the \`DEBUG\` means is that the above will apply also for control statement commands.

Reply:
Not sure it is needed here tracing is on and it should be enough

Reply:
JFYI, the install refactor was moved to a separate PR https://github.com/ZacWalk/mingw-woarm64-build/pull/34